signac/contrib/project.py

```diff
@@ -658,6 +668,14 @@ def groupby(self, key=None, default=None):
             for key, group in project.groupby('a'):
                 print(key, list(group))
 
+            # Group jobs by document value 'a'.
+            for key, group in project.groupby('doc.a'):
+                print(key, list(group))
+
+            # Group jobs by jobs.sp['a'] and job.document['b']
+            for key, group in project.groupby('a', 'doc.b'):
+                print(key, list(group))
+
```

Comment:
This looks like it's left-over from debugging.

Reply:
I updated the doc-string of the `groupby` function and added examples showing how to use doc key.

Reply:
Got it. That was hard to see in the GitHub diff view.